stories/object-marker/custom.scss

```diff
@@ -0,0 +1,3 @@
+.fd-object-marker {
+  margin-right: 20px;
```

Comment:
can we use rem here. Also the measurements are incorrect, I  think you should use :after instead of before since the space is usually after the marker and add extra spacing if after that icon, there is another

Reply:
I think he is using this just to add some spacing between the object markers in the documentation, it's not part of the component styling